specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/registeredidentities.json

```diff
@@ -284,18 +294,22 @@
       "type": "object",
       "properties": {
         "name": {
-          "type": "string"
+          "type": "string",
+          "readOnly": true

```

Comment:
None of these properties seem to have descriptions, which is something the linter is supposed to be looking for. Is there a reason descriptions are not included?

Reply:
Thank-you for letting me know. We should definitely have descriptions in this case.